bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -1061,4 +1078,17 @@ private void setContentAssistsEnablement(boolean enable) {
 	private void updateContentAssistAvailability() {
 		setContentAssistsEnablement(findReplaceLogic.isAvailableAndActive(SearchOptions.REGEX));
 	}
+
+	public void decorate() {
```

Comment:
Why `public` and not `private`?

Reply:
first the idea was to make an interface from which all three classes inherit in which the decorate method is made. I discarded the canges because it would have been as helpful as initially thought.

Reply:
done